torch/distributed/fsdp/flat_param.py

```diff
@@ -152,6 +154,9 @@ class FlatParameter(nn.Parameter):
             ``_numels``, ``_shapes``, and ``_prefixed_param_names``.
         _shared_param_infos (Tuple[SharedParamInfo, ...]): Shared parameter
             info entries; see :class:`SharedParamInfo`.
+        _param_extensions (Tuple[Optional[Any], ...]): Parameter extensions
```

Comment:
nit: Maybe mention this is experimental and users should not depend on it being around?